apps/livreBatch/livre_batch.py

```diff
@@ -104,7 +104,7 @@ def _build_sbatch_script(self, start, end):
         values = self.dict
         values['start'] = start
         values['end'] = end
-        values['num_frames'] = end - start + 1
+        values['num_frames'] = end - start
```

Comment:
This means the the frame interval given from the command line should be understood as open on the right, shouldn't it? Perhaps you should document that.